server/data/apps.json

```diff
@@ -27,6 +27,53 @@
       }
     ]
   },
+  {
+    "appName": "AiPodNav",
+    "appType": [
+      "podcast player",
+      "website",
+      "directory",
+      "other"
+    ],
+    "appUrl": "https://www.aipodnav.com/",
+    "appIconUrl": "AIPodNav.png",
+    "platforms": [
+        "Mobile",
+        "Web",
+        "Windows",
+        "macOS"
+    ],
+    "supportedElements": [
+      {
+        "elementName": "directory",
+        "elementURL": "https://www.aipodnav.com/home"
+      },
+      {
+        "elementName": "charts",
+        "elementURL": "https://www.aipodnav.com/discover/best-podcasts"
+      },
+      {
+        "elementName": "mind map",
+        "elementURL": "https://www.aipodnav.com/"
+      },
+      {
+        "elementName": "summary",
+        "elementURL": "https://www.aipodnav.com/discover/new-ai-podcast-summary"
+      },
```

Comment:
Hi, this are not Podcasting 2.0 tags

Reply:
Alright, I have resubmitted it.

Reply:
I have took a look at your project, but I don't find any reference to the `episodes tag` in the link you provided

In what way are you using it?
https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#episode